example/build.yaml

```diff
@@ -0,0 +1,9 @@
+# This file is only needed if you want to tweak the built_value generator
+# options. The only available option is "header", which sets the header of
+# each generated file.
+targets:
+  $default:
+    builders:
+      built_value_generator|built_value:
+        options:
+          header: // This is generated code; please do not modify by hand.
```

Comment:
I think a multiline example like shown in https://github.com/dart-lang/json_serializable/blob/master/example/build.yaml would be better
I think it's more common and I didn't know before I saw that that YAML allows to do it this way

Reply:
Thanks for the suggestion. When I looked it up

https://yaml-multiline.info/

I found it's pretty complicated. I'll add a link to that page rather than an example.

Reply:
great